docs/1.2-DRAFT/metadata.md

```diff
@@ -151,6 +151,22 @@ To support geometry in [Places](contextual-entities.md#places), these terms from
 {: .note }
 > In this specification the proposed Bioschemas terms use the temporary <https://bioschemas.org/> namespace; future releases of RO-Crate may reflect mapping to the <http://schema.org/> namespace.
 
+From [CodeMeta](https://w3id.org/codemeta#):
+
+* `buildInstructions` mapped to <https://w3id.org/codemeta#buildInstructions>
+* `developmentStatus` mapped to <https://w3id.org/codemeta#developmentStatus>
+* `continuousIntegration` mapped to <https://w3id.org/codemeta#continuousIntegration>
+* `embargoEndDate` mapped to <https://w3id.org/codemeta#embargoEndDate>
+* `hasSourceCode` mapped to <https://w3id.org/codemeta#hasSourceCode>
+* `isSourceCodeOf` mapped to <https://w3id.org/codemeta#isSourceCodeOf>
+* `issueTracker` mapped to <https://w3id.org/codemeta#issueTracker>
+* `readme` mapped to <https://w3id.org/codemeta#readme>
+* `referencePublication` mapped to <https://w3id.org/codemeta#referencePublication>
+* `softwareSuggestions` mapped to <https://w3id.org/codemeta#softwareSuggestions>
```

Comment:
Same as above. The current namespace is https://codemeta.github.io/terms/, even if that one is not machine-readable